generator/src/main/java/com/scottlogic/deg/generator/inputs/InvalidProfileException.java

```diff
@@ -5,4 +5,8 @@
     public InvalidProfileException(String message) {
         super(message);
     }
+
+    public InvalidProfileException(Exception e) {
```

Comment:
See other comments RE one exception type, if this is adopted then this constructor can be removed.